entity-service/src/main/resources/db/liquigraph/changelog.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<changelog>
+    <changeset id="migrate-temporal-properties-to-datetime-data-type" author="stellio-neo4j-migrations">
+        <query>MATCH (n) set n.modifiedAt = dateTime(n.modifiedAt)
```

Comment:
what will happen if one of property (createdAt, modifiedAt, observedAt) does not exist ?

Reply:
`MATCH(n)` will match all the nodes in the database ... which I guess should be OK ... but it can be dangerous, always safer to clearly define your targets (e.g. all nodes labelled 'Entity', 'Property' or 'Relationship')

Reply:
if a property does not exist it will be ignored.